test/unit/workflows/test_workflow_progress.py

```diff
@@ -100,7 +100,7 @@ def _set_previous_progress(self, outputs):
 
             workflow_invocation_step_state = model.WorkflowRequestStepState()
             workflow_invocation_step_state.workflow_step_id = step_id
-            workflow_invocation_step_state.value = True
```

Comment:
We have to fix these column definitions. This is just a wild line of code.

Reply:
Please merge https://github.com/galaxyproject/galaxy/pull/17902 and follow up with a proper type definitions for that column. The truth is it can be any JSON serializable type - it is sort of up to workflow module to process it - so maybe Any or maybe start with ``bool`` and create a comment somewhere that it will need to be unioned with other types if we add more workflow module types. I'm not sure if conditionals use this or not - probably worth looking into but maybe the typing alone would tell us.

Reply:
Will do.